pkgs/applications/video/lossless-cut-bin/linuxbin.nix

```diff
@@ -0,0 +1,83 @@
+# Binary currently ususable (but is build successfully).
```

Comment:
Is this statement still true?

Reply:
I tried but it doesn't work.
I'll remove it.